zerver/lib/actions.py

```diff
@@ -825,6 +882,7 @@ def check_stream_name(stream_name):
         raise JsonableError("Invalid stream name")
 
 def send_pm_if_empty_stream(sender, stream, stream_name):
+    # type: (UserProfile, Optional[Stream], text_type) -> None
```

Comment:
I think this function doesn't actually support stream=None, see the second if statement below...